### PR TITLE
Update debugging_crashes.md

### PR DESCRIPTION
**Pull request by asklar**
Adds instructions for collecting and auto-analyzing crash dumps using `analyze-crash.ps1`

### CODE REVIEW

File: docs/debugging_crashes.md [hunk 1]
```diff
@@ -34,6 +34,16 @@ For example, running these commands in an administrative command prompt will set
 
 After you repro the crash, you should find a dump in the specified folder.
 
+You can automate crash dump collection and filing bugs by following these steps:
+1. Download the script at https://aka.ms/RNW/analyze-crash.ps1, for example to `C:\temp`
+2. Open an admin PowerShell. If you haven't enabled running unsigned scripts yet, do that by running: `Set-ExecutionPolicy Unrestricted`
+3. Run the script and pass it the name of your app's exe: `C:\temp\analyze-crash.ps1 -ExeName MyApp -Repo microsoft/microsoft-ui-xaml`
```
Comment (codendone) [inline docs/debugging_crashes.md:40]: >MyApp [](http://example.com/codeflow?start=94&length=5)

Just to confirm, this is just "MyApp" not "MyApp.exe"?

File: docs/debugging_crashes.md [hunk 2]
```diff
@@ -34,6 +34,16 @@ For example, running these commands in an administrative command prompt will set
...
+The script will set up automatic crash dump collection for your app, download the native debugging tools (including the command line debugger cdb), and ask you to reproduce the crash.
```
Comment (codendone) [inline docs/debugging_crashes.md:41]: >command line [](http://example.com/codeflow?start=120&length=12)

nit: add hyphen for "command-line"?

**Review by codendone — APPROVED** [review]
:shipit:

File: docs/debugging_crashes.md (hunk 1 above)
Comment (asklar) [reply docs/debugging_crashes.md:40]: correct, the script appends .exe to the `LocalDumps` key

File: docs/debugging_crashes.md (hunk 2 above)
Comment (asklar) [reply docs/debugging_crashes.md:41]:
```suggestion
The script will set up automatic crash dump collection for your app, download the native debugging tools (including the command-line debugger cdb), and ask you to reproduce the crash.
```

**Pull request merged**